docs/guides/proxysql/clustering/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: ProxySQL Clustering
+menu:
+  docs_{{ .version }}:
+    identifier: guides-proxysql-clustering
+    name: ProxySQL Clustering
+    parent: guides-proxysql
```

Comment:
There is no identifier named `guides-proxysql` in the parent structure.  It will be `prx-proxysql-guides` as defined here https://github.com/kubedb/docs/blob/proxysql-doc-2/docs/guides/proxysql/_index.md .

Reply:
This same changes is applicable for all the folders under `docs/guides/proxysql`.